src/components/AddNewOrganizationModal.tsx

```diff
@@ -186,39 +200,41 @@ export default function AddNewOrganizationModal(props: AddNewOrganizationModalPr
             value={newOrganization.description}
           />
         </Grid>
-        {timeZonesEnabled && (
-          <Grid item xs={12}>
-            <TimeZoneSelector
-              label={strings.TIME_ZONE_REQUIRED}
-              onTimeZoneSelected={onTimeZoneChange}
-              selectedTimeZone={newOrganization.timeZone}
-              tooltip={strings.TOOLTIP_TIME_ZONE_ORGANIZATION}
-              errorText={timeZoneError}
-            />
-          </Grid>
-        )}
         <Grid item xs={12}>
           <Select
```

Comment:
@constanzauanini i will change these to autocomplete as well.. i'm wondering if we should just have a component that does the country/state selection since it's redundant.. i can do that after you merge your PR